### PR TITLE
add backend arg

### PR DESCRIPTION
**Pull request by xvjiarui**

### CODE REVIEW

File: mmcv/image/io.py [hunk 1]
```diff
@@ -131,10 +131,17 @@ def imread(img_or_path, flag='color', channel_order='bgr'):
             candidates are `color`, `grayscale` and `unchanged`.
             Note that the `turbojpeg` backened does not support `unchanged`.
         channel_order (str): Order of channel, candidates are `bgr` and `rgb`.
+        backend (str|None): The image decoding backend type. Options are `cv2`,
+            `pillow`, `turbojpeg`, `None`. If backend is None, the global
+            imread_backend will be used.
```
Comment (Johnson-Wang) [inline mmcv/image/io.py:136]: Default value.

File: mmcv/image/io.py [hunk 2]
```diff
@@ -165,22 +172,29 @@ def imread(img_or_path, flag='color', channel_order='bgr'):
...
 
     Returns:
         ndarray: Loaded image array.
     """
-    if imread_backend == 'turbojpeg':
+
+    if backend is None:
+        backend = imread_backend
+    assert backend in supported_backends
```
Comment (Johnson-Wang) [inline mmcv/image/io.py:191]: raise ValueError instead of using assert.

File: mmcv/image/io.py (hunk 1 above)
Comment (Johnson-Wang) [inline mmcv/image/io.py:136]: Change the last line to:
If backend is None, the global imread_backend specified by \`mmcv.use_backend\` will be used.

File: mmcv/image/io.py [hunk 3]
```diff
@@ -131,10 +131,20 @@ def imread(img_or_path, flag='color', channel_order='bgr'):
             candidates are `color`, `grayscale` and `unchanged`.
             Note that the `turbojpeg` backened does not support `unchanged`.
         channel_order (str): Order of channel, candidates are `bgr` and `rgb`.
+        backend (str|None): The image decoding backend type. Options are `cv2`,
+            `pillow`, `turbojpeg`, `None`. If backend is None, the global
+            imread_backend specified by `mmcv.use_backend` will be used.
```
Comment (hellock) [inline mmcv/image/io.py:136]: \`\`mmcv.use_backend()\`\`

**Pull request merged**